### PR TITLE
Makes menu sticky

### PR DESCRIPTION
**Pull request by husnimun**
Hi! This site is really awesome and helpful!

I added some styles to make the navigation menu stick to the top so it's easier to navigate the content. What do you think about it? :smiley:

### CODE REVIEW

**Comment by melanieseltzer** [comment]
I love it! One small thing (and I'm not even sure there's a solution) - I'm noticing a weird issue in Safari on Mac, where the ability to scroll inside the nav seems to disappear randomly. Any ideas? I can't seem to find anything online about it other than this [question](https://stackoverflow.com/q/48676086), but sadly no solutions...

kapture 2018-05-27 at 10 11 34

**Comment by husnimun** [comment]
I found a [question](https://stackoverflow.com/questions/35469227/overflow-y-not-working-in-safari-inside-a-modal/35560173) about this issue but I can't verify the solution since I don't have a mac. It seems safari has a bug with `overflow: auto`.

Does this issue always happen with `overflow: auto` in safari?

**Comment by melanieseltzer** [comment]
Tbh I don't use Safari either, just checked this PR on there to be thorough. But it does indeed seem like a universal bug of sorts. MDN examples for overflow auto/scroll behave the same way on Safari for me, so not just this project. Let's just proceed since it doesn't seem like we can do anything 👍

File: src/styles/custom.scss [hunk 1]
```diff
@@ -29,6 +29,15 @@ pre,
   text-align: center;
 }
 
+.sticky-menu {
+  @media screen and (min-width: 768px) {
```
Comment (melanieseltzer) [inline src/styles/custom.scss]: This will need to be min-width: 769px to match Bulma's column breakpoint.

Other than that, good to go 👍

Comment (husnimun) [reply src/styles/custom.scss]: I've updated it :+1:

**Comment by melanieseltzer** [comment]
Merging 🎉 Thanks for contribution!!

**Pull request merged**